src/main/java/com/solr/clientwrapper/usecase/solr/core/CreateSolrCore.java

```diff
@@ -1,27 +1,28 @@
 package com.solr.clientwrapper.usecase.solr.core;
 
```

Comment:
Avoid interfering and committing already working layers/classes **unless required**.